404.php

```diff
@@ -25,7 +25,9 @@ function add_body_classes( $classes ) {
 
 $context = Timber::get_context();
 
-$context['page_notfound_image']       = esc_url( get_template_directory_uri() . '/images/404-header.jpg' );
+$theme_uri = is_child_theme() ? get_stylesheet_directory_uri() : get_template_directory_uri();
```

Comment:
Why do you need to check if this code is in the child theme, since you are running it from inside the child theme?

Reply:
Also, will get_template_directory_uri return the child theme directory, or the parent theme?

Reply:
This code is inside the master theme and it will run unless there is a 404.php file inside the child theme.
So, if a child theme is active (e.g. Greek site) then $theme_uri will get the value of the child theme uri (get_stylesheet_directory_uri)
else if master theme is active (e.g. International site) it will get the value of the parent theme uri (get_template_directory_uri)

Reply:
> This code is inside the master theme
I don't understand the above phrase. You have committed this code in this specific child theme.

Additionally: no site has the master theme active. All sites have their child theme active. 
So, this code makes no sense. If you run this code in the child theme, this code is getting run only if this specific child theme is active, in which case the whole question is irrelevant.

Reply:
@koyan 
Actually, you were right on this one, there was no need to check if a child theme is active since always the child theme is active.
It would work simply by changing get_template_directory_uri() to get_stylesheet_directory_uri() in order to get the child theme uri and then add the images/404-header.jpg in the the planet4-child-theme